client/CMakeLists.txt

```diff
@@ -51,13 +51,21 @@ set(trillek-client_SOURCES ${trillek-client_SOURCES} PARENT_SCOPE) # so tests ca
 
 # macOS needs to link libobjc
 set(OSX_LIBRARIES "")
-if (APPLE)
+
+
+IF (APPLE)
+EXEC_PROGRAM(uname ARGS -v OUTPUT_VARIABLE DARWIN_VERSION)
```

Comment:
Just a small nit-pick, but can you indent these some to match the rest of the file? Also there appears to be an extra newline on line 54.

Reply:
Sure, on it.

Reply:
Looks good